docs/working-drafts/issue-29-erratum.md.adoc

```diff
@@ -0,0 +1,135 @@
+// please start a new sentence on a new line.
+// this facilitates commentary on this document using code-review tools.
+
+= Issue 29 - dfdl:prefixLengthKind Simple Type having DFDL Statement Annotations
+
+
+Sections of the DFDL v1.0 Spec which will require updating include:
+
+## Section 9.5 - Evaluation Order for Statement Annotations
+
+An element (or element reference) having lengthKind 'prefixed' can
+have statements associated with its prefix length type. This is
+discussed below in Section (new sub-section of 9.5).
+
+A new sub-section is added: Evaluation Order for Statement Annotations
+for Elements with dfdl:lengthKind 'prefixed'.
+
+This new sub-section contains the content below.
+
+When an element or element reference has lengthKind 'prefixed' there
+can be additional statement annotations associated with the prefix
+length type.
+
+Definition: the resolved set of statement annotations for the simple
+type that is identified by the dfdl:prefixLengthKind property are
+called the _prefix statement annotations_.
+
+Definition: the resolve set of statement annotations for the nested
+prefix simple type that is identified by the dfdl:prefixLengthKind
+property on a simple type itself having lengthKind 'prefixed', are
+called the _prefix-prefix statement annotations_.
+
+For statement annotations associated with any prefix length, any DFDL
+expressions evaluated cannot refer to the DFDL Infoset using path
+expressions.
+
+The value of the prefix (or prefix-prefix) itself can be referenced
+using the expression ".".
+
+When parsing the value of the "." expression for a prefix (or
+prefix-prefix) quasi-element is as before any adjustment (subtraction)
```

Comment:
"quasi-element" is a Daffodil term, not used in the spec so far. IBM DFDL calls them synthetic elements I think.

Reply:
This definition may want to go with the definition of prefix lengthKind, not necessarily in this section on evaluation order.